developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx

```diff
@@ -2,13 +2,13 @@
 title: Getting hands-on
 ---
 
-### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;Preparation
+### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp; Preparation
 
 :::note Obtain a free trial OpenAI API key
-This section includes queries using the OpenAI inference endpoint. So, we recommend obtaining an OpenAI account, and obtaining a **free API key which should be sufficient** for all of the below.
+This section includes queries using the OpenAI inference endpoint. If you don't already have an OpenAI account, we recommend getting an account with a **free API key** The free key should be sufficient for these exercises.
```

Comment:
Hmm. I do not think they do free keys any more unfortunately.

Let's just rephase it as follows:

`This section includes queries using an OpenAI inference endpoint. If you don't already have an OpenAI account, we recommend creating one. At the time of writing, OpenAI provides trial credits which should be sufficient for these exercises.`

Reply:
Bummer! 

Updated